packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -33,6 +34,9 @@ export interface ITruncatedFormatProps extends IProps {
      */
     detectTruncation?: boolean;
 
+    parentCellHeight?: string;
+    parentCellWidth?: string;
```

Comment:
docs? are these props even necessary here? you cast to `React.ReactElement<any>` above

Reply:
See comments on PR -- this is explicitly necessary for the shouldComponentUpdate -- we don't actually use them, but we need them to keep from rerendering and remeasuring every time, without knowing if the size has changed.